### PR TITLE
Miscellaneous changes

### PR DESCRIPTION
**Pull request by dalg24**

### CODE REVIEW

File: docs/source/install.rst [hunk 1]
```diff
@@ -80,25 +80,17 @@ Create a ``do-configure`` script such as:
...
-        -D TPL_ENABLE_BinUtils:BOOL=OFF \
-        -D Trilinos_ENABLE_ALL_OPTIONAL_PACKAGES:BOOL=OFF \
+        -D CMAKE_BUILD_TYPE=Release \
+        -D TPL_ENABLE_MPI=ON \
+        -D TPL_ENABLE_BLAS=ON \
+        -D TPL_ENABLE_LAPACK=ON \
+        -D TPL_ENABLE_Boost=ON \
+        -D Trilinos_ENABLE_ALL_OPTIONAL_PACKAGES=OFF \
         -D Trilinos_EXTRA_REPOSITORIES="DataTransferKit" \
-        -D Trilinos_ENABLE_EXPLICIT_INSTANTIATION:BOOL=ON \
```
Comment (aprokop) [inline docs/source/install.rst]: Why remove this? People will try compile without ETI and will run away screaming.

Comment (dalg24) [reply docs/source/install.rst]: Because I did not want to bother giving the a list of types to instantiate.

Comment (dalg24) [reply docs/source/install.rst]: I added a note that the `do-configure` script might not be the most up-to-date place to look.

@aprokop Would you add the explicit instantiation lines to the script?

Comment (aprokop) [reply docs/source/install.rst]: @dalg24 Which script?

Comment (dalg24) [reply docs/source/install.rst]: "do-configure" in `docs/source/install.rst`

Comment (aprokop) [reply docs/source/install.rst]: Done.

**Comment by dalg24** [comment]
Thanks @aprokop

**Pull request merged**